source/fundamentals/connection/connect.txt

```diff
@@ -30,25 +30,25 @@ connect to a MongoDB deployment. It instructs the driver on how it should
 connect to MongoDB and how it should behave while connected. The following
 example shows each part of the connection URI:
 
-.. figure:: /includes/figures/connection-string-parts.png
+.. figure:: /includes/figures/dns_seedlist_connection_string_parts.png
    :alt: Each part of the connection string
 
-In this example, we use ``mongodb`` for the protocol, which specifies
-the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.
-
-If your instance or deployment has a DNS SRV record, you can use the
+In this example, we connect to an Atlas MongoDB deployment that has a 
+DNS SRV record. For more details, refer to the 
 :manual:`DNS Seed List Connection Format
-</reference/connection-string/#dns-seed-list-connection-format>` for
-your connection string. This format offers more flexibility of
-deployment and the ability to change the servers in rotation without
-reconfiguring clients.
+</reference/connection-string/#dns-seed-list-connection-format>` 
+documentation. This format offers flexibility in deployment and the 
+ability to change the servers in rotation without reconfiguring clients.
 
 .. note::
 
    To learn how to retrieve your connection string in Atlas, see the
    :atlas:`Atlas driver connection guide </driver-connection>`.
 
-The next part of the connection string contains your credentials
+Alternately, you could use ``mongodb`` for the protocol, which specifies
+the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.
```

Comment:
Suggestion:

I think it could be good to mention that you must use it when you aren't connecting to a DNS SRV address


```suggestion
If you are connecting to an instance or replica set that does not have a DNS SRV address, you must use the ``mongodb`` protocol, which specifies
the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.
```